compiler/rustc_codegen_ssa/src/back/write.rs

```diff
@@ -2100,28 +2096,25 @@ impl<B: ExtraBackendMethods> OngoingCodegen<B> {
 }
 
 pub(crate) fn submit_codegened_module_to_llvm<B: ExtraBackendMethods>(
-    _backend: &B,
-    tx_to_llvm_workers: &Sender<Box<dyn Any + Send>>,
+    coordinator: &Coordinator<B>,
```

Comment:
Any specific reason to pass full coordinator while only the sender is actually used?
(Here and below.)

Reply:
The Message enum is not public, so `Sender<Message>` would complain about a private type in a public interface.